pkg/packagekit/updates.less

```diff
@@ -32,20 +32,20 @@
 }
 
 /* override default cockpit CSS, as this doesn't fit our table */
-table.listing-ct {
+.listing-ct {
     margin-top: 0;
     width: 100%;
 }
 
-table.listing-ct thead th {
+.listing-ct thead th {
     padding-left: 0;
 
     &:last-child {
         text-align: left;
     }
 }
 
-tr.listing-ct-item {
+.listing-ct tr.listing-ct-item {
```

Comment:
Everything from here on upward in this file is from an older commit I squashed together, where I made the list overrides generic. As it dealt with responsiveness, I realized it was actually important, so I re-added it. The changes above are legacy.